tests/Integration/RedirectsTest.php

```diff
@@ -25,11 +37,6 @@ public function get_redirect_data() {
 				'http://example.com',
 			),
 
-			'redirect_unicode_in_path'  => array(
-				// https://www.w3.org/International/articles/idn-and-iri/
-				'/JP納豆',
-				'http://example.com',
-			),
 		);
 	}
 
```

Comment:
The `tests/Integration/bootstrap.php` file below has apparently only got 1 addition, but is too large to be shown?!

Reply:
It was a missing line, in the end, should be fixed now as well.